src/lib/rules/performance-budget/connections.ini

```diff
@@ -0,0 +1,91 @@
+# Original file from WebPageTest (https://github.com/WPO-Foundation/webpagetest/blob/master/www/settings/connectivity.ini.sample)
```

Comment:
This might be a stupid question: Why not just use `json` file instead of a text file, then we don't need the parser?

Reply:
I thought about it, but because this file comes from WebPageTest I'd like to avoid having to reformat it when there's an update. This way we can potentially update it dynamically the same way we do with the no vulnerable libraries rule.